election/models.py

```diff
@@ -1,38 +1,57 @@
 from django.db import models, IntegrityError
 from django.contrib.postgres.fields import ArrayField
 from libs.django_randomprimary import RandomPrimaryIdModel
+import logging
 
+logger = logging.getLogger(__name__)
 
-NUMBER_OF_MENTIONS = 7
+MAX_NUM_GRADES = 20
```

Comment:
20 reflects the book of Balinski and Rida. A large bulk of their examples is done with grades from 0 to 20. I use this number as a default value as it was before, but I agree it would be better to delete it completely. I just think this is out of the scope of this PR

Reply:
the argument "it is in the Rida's book is fine for me :smile: